osbenchmark/worker_coordinator/runner.py

```diff
@@ -1925,9 +1937,11 @@ class Composite(Runner):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.supported_op_types = [
-            "open-point-in-time",
-            "close-point-in-time",
+            "create-point-in-time",
+            "delete-point-in-time",
+            "get-point-in-time",
             "search",
+            "paginated-search",
```

Comment:
Where it the code for this ?

Reply:
The code is already present in the code base